frontend/src/app/coworking/ambassador-home/ambassador-home.component.css

```diff
@@ -1,7 +1,20 @@
 .mat-mdc-card {
-    max-width: 100%;
+  max-width: 100%;
+}
+
+.mat-mdc-card-header {
+  margin-bottom: 16px;
+}
+
+.walkinReservation.mat-mdc-card-content:last-child {
+  padding-bottom: 0;
+}
+
+coworking-dropin-availability-card:last-child {
+  display: block;
+  margin-bottom: 16px;
```

Comment:
Nit: This margin did not seem to be active on my computer!

Reply:
Fair enough, removing for now